kits/repo-interview-prep/README.md

```
@@ -0,0 +1,109 @@
+    "concepts_to_review": [...],
+    "red_flags": [...],
+    "strengths_to_highlight": [...]
+  }
+}
+```
+
+---
+
+## Flow Architecture
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)